src/hourly-operating-workspace/hourly-operating.service.ts

```diff
@@ -52,6 +52,9 @@ export class HourlyOperatingWorkspaceService {
         this.matsMonitorHourlyValueService.export(
           hourlyOperating?.map(i => i.id),
         ),
+        this.matsDerivedHourlyValueService.export(
+          hourlyOperating?.map(i => i.id),
+        ),
```

Comment:
There's no need to keep repeating the same loop on these, it'd be better to set a constant.

```
const hourlyOperatingIds = hourlyOperating.map(hourlyOperatingDatum => {
    return hourlyOperatingDatum.id;
})
```

Emissions service does the same for locationIds.